Documentation/ColumnsConfig/Type/Inline/Index.rst

```diff
@@ -11,39 +11,43 @@ IRRE / inline
     column type :ref:`file <columns-file>` instead.
 
 ..  versionadded:: 13.0
-    When using the `inline` type, TYPO3 takes care of
-    :ref:`generating the according database field <t3coreapi:auto-generated-db-structure>`.
-    A developer does not need to define this field in an extension's
+    When using the `inline` type, TYPO3
+    :ref:`generates the correct database fields <t3coreapi:auto-generated-db-structure>`.
+    Developers do not need to define this field in an extension's
     :file:`ext_tables.sql` file.
 
-Inline-Relational-Record-Editing (IRRE) offers a way of directly editing parent-child-relations in one backend view.
-New child records are created using AJAX calls to prevent a reload of the complete backend view.
-
-type='inline' is a powerful element that allows handling a huge list of relations. This includes simple :code:`1:n`
-and nested :code:`1:n-1:n` relations, as well as various :code:`m:n` relation scenarios with different view aspects
-and localization setups. Combined with other TCA properties in 'ctrl' and 'types', the sheer amount of different
-display variants is amazing.
-
-Be aware that inline has been mostly designed to manage :code:`1:n` relations. In those relations one parent record
-can have multiple children, but one child is connected to only one parent. Children can not be moved from one parent
-to another.
-
-The exception are :code:`m:n` relations where a child can be connected via an intermediate table to
-multiple parents, and the intermediate table can have editable fields on its own, thus attaching additional information
-to one specific parent-child relation. In the core, the "FAL" / resource handling is an example of that. A parent record
-(for instance of table "tt_content") is connected via table "sys_file_reference" to one media resource in "sys_file".
-A sys_file record has table "sys_file_metadata" as child record which holds meta information details of the file in
-question (for instance a description). This information can be overwritten for the specific file resource used in
-"tt_content" by adding a new description in table "sys_file_reference". The various inline and field properties
-like "placeholder" help managing this complex setup in TCA.
+Inline-Relational-Record-Editing (IRRE) is a way of editing parent-child-relations in the backend view.
+Instead of child records already having to exist, new child records are created
+using AJAX calls (to prevent reloading the complete backend view).
+
+The type='inline' is a powerful element that can handle many types of relation,
+including simple :code:`1:n` and nested :code:`1:n-1:n` relations, aswell as :code:`m:n`
```

Comment:
Really "aswell"? Merriam-Webster suggests "as well":
https://www.merriam-webster.com/dictionary/as%20well#phrases-anchor

Reply:
Both pons and google translate recognize aswell. We should trust a native speaker here

Reply:
@linawolf @sarahmccarthy123 Merrian-Webster says only "as well", this is the dictionary mentioned as reference in the Howto document. But well, I can leave the review altogether then in the future as a native speaker is sacrosanct and no discussion about it is allowed.

Reply:
Hello yes I looked it up and it turns out that "aswell" is an archaic spelling of "as well" and that it's a common error. Maybe it's because of all the classic literature that I like reading... :-)

Reply:
@linawolf So Lina, your turn now, as you approved it.

Reply:
:-) I don't mind it being changed to "aswell" :-). Merriam-Webster is American though so if you had said it was in the Oxford Dictionary I would have been more convinced ;-)